migrator/src/main/java/uk/nhs/digital/ps/migrator/config/ExecutionConfigurator.java

```diff
@@ -155,20 +143,20 @@ private Path getPathArg(final ApplicationArguments args, final String commandLin
                     " NOTE that the directory is deleted and re-created on each run."
             ),
             describe(
-                TAXONOMY_SPREADSHEET_PATH,
-                "Path to the spreadsheet that contains the taxonomy we want to import into hippo." +
-                    " Optional - if not provided, taxonomy will not be imported."
+                TAXONOMY_DEFINITION_IMPORT_PATH,
+                "Path to the spreadsheet that contains the taxonomy definition we want to import into hippo." +
+                    " Required if --" + NESSTAR_CONVERT_FLAG + " is specified, optional otherwise."
             ),
             describe(
-                TAXONOMY_OUTPUT_PATH,
-                "Path to output the taxonomy JSON to import into hippo." +
+                TAXONOMY_DEFINITION_OUTPUT_PATH,
+                "Path to output the taxonomy definition JSON to import into hippo." +
                     " Optional - if not provided, one will be created in a temporary space."
             ),
             describe(
                 TAXONOMY_MAPPING_IMPORT_PATH,
-                "Path to taxonomy yaml import file used for the mapping of keywords in the migrator process." +
-                    " Optional - if not provided, the sample (partially populated) taxonomy file from" +
-                    " the development module will used instead."
+                "Path to taxonomy mapping spreadsheet used to map migrated datasets' P code to " +
+                    "hippo taxonomy keys as imported from the taxonomy definition." +
+                    " Required if --" + NESSTAR_CONVERT_FLAG + " is specified."
```

Comment:
Is this mandatory too based on assertRequiredArgs?

Reply:
Yes in the nesstar convert task the following are asserted:
`assertRequiredArgs(hippoImportDir, nesstarUnzippedArchiveLocation, taxonomyDefinitionImportPath, taxonomyMappingImportPath);`